_integration-schemas/netsuite/custom_record.md

```diff
@@ -26,7 +28,7 @@ attributes:
     description: |
       The time the record was last updated.
 
-      **Note**: This field will be available only if the custom record type has **Show Last Modified** or **Show Creation Date** enabled in {{ integration.display_name }}. Refer to the [Custom records](#custom-records) section for more info.
+      **Note**: This field will be available only if the custom record type has **Show Last Modified** enabled in {{ integration.display_name }}. Refer to the [Custom records](#custom-records) section for more info.
```

Comment:
Shoot, I missed this part. This field will always be available, but Netsuite won't return values for it without the checkbox checked. Came with Creation Date.

Creation Date will aslways be selectable, but may not have values if that checkbox is checked.

Reply:
Got it! I'll make the change.